theurgy.lic

```diff
@@ -776,7 +782,11 @@ class TheurgyActions
     return false unless holy_water?
     safe_walk_to @data['altar']['id']
     DRCI.get_item_safe(@water_holder, @theurgy_supply_container)
-    fput 'clean altar with holy water'
+    case bput('clean altar with holy water','Roundtime','You are a bit too busy performing to do that')
+    when 'You are a bit too busy performing to do that'
```

Comment:
Doesn't `bput` already handle this?

See https://github.com/rpherbig/dr-scripts/blob/master/common.lic#L164

Reply:
Yup!  I forgot entirely.  I just get mega annoyed when somehow I'm still playing a song when I get to the altar and pause forever because fput/waitfor is broken by obnoxious cowbell.  Fixed!